frameworks/solid/src/benchmark.jsx

```diff
@@ -232,24 +226,23 @@ const Benchmark = () => {
     console.log('starting benchmark');
     setTimeout(runBenchmark, 1000);
 
-    return (<Show when={data()}>
+    return (<Show when={data().length}>
         <View ref={container}>
-            <For each={ data() }>{ (row) => {
-                return <View x={row.x} y={row.y} width={row.width} height={row.height} color={row.color}>
-                    <Text 
-                        x={5}
-                        y={2}
-                        width={row.width}
-                        height={row.height}
-                        alpha={0.8}
-                        fontFamily={"Ubuntu"}
-                        color={row.textColor}
-                        fontSize={row.fontSize}>
-                        {row.label}
-                    </Text>
-                </View>
-            }}
-            </For>
+          <Index each={data()}>{(row) => (
+                    <view x={row().x} y={row().y} width={row().width} height={row().height} color={row().color}>
```

Comment:
Youre using `View` all over your example app - similar for `Text` vs `text` ... sounds like this is not respective of what you would use in a real app.

Reply:
Updated